### PR TITLE
Quorum doc

### PR DESCRIPTION
**Pull request by itisravi**
This is with reference to https://bugzilla.redhat.com/show_bug.cgi?id=1663337

### CODE REVIEW

**Comment by itisravi** [comment]
@pranithk @karthik-us  Please review.

File: docs/Administrator Guide/arbiter-volumes-and-quorum.md [hunk 1]
```diff
@@ -194,13 +195,6 @@ Say B1, B2 and B3 are the bricks:
 that B2 and B3's pending xattrs blame each other and therefore the write is not
 allowed and is failed with an EIO.
 
-There is a corner case even with replica 3 volumes where the file can
```
Comment (karthik-us) [inline docs/Administrator Guide/arbiter-volumes-and-quorum.md]: Also remove the mention of this corner case @213

Comment (itisravi) [reply docs/Administrator Guide/arbiter-volumes-and-quorum.md]: @karthik-us  Done.

**Comment by karthik-us** [comment]
LGTM.

**Comment by itisravi** [comment]
@harigowtham Please review and merge if it looks okay.

**Pull request merged**